docs/clients/grpc/getting-started/connecting.md

```diff
@@ -9,7 +9,9 @@ Install the client SDK package to your project.
 
 ```
 $ dotnet add package EventStore.Client.Grpc.Streams --version 20.6.1
+$ dotnet add package Grpc.Net.Client --version 2.32.0
 ```
+<!-- TODO: when https://github.com/EventStore/EventStore/issues/2707 is resolved and new version with the fix is released - remove the manual Grpc.Net.Client installation -->
```

Comment:
I would add such information as well instead of being a comment.

Reply:
@yordis I'd agree - if that' was the end solution. We assume that within one month we'll have the version that won't require this additional step.
I added this step as for now give explicit information for new users.
Thoughts?

Reply:
I would still prefer linking to Microsoft docs
https://docs.microsoft.com/en-us/aspnet/core/grpc/troubleshoot?view=aspnetcore-5.0#call-insecure-grpc-services-with-net-core-client

Because it's only needed when connecting to an insecure instance.

Reply:
If we know an already an easy solution, we should suggest it without the redirections (especially that's only temporary until the new version is released). I think that this part of the documentation helps someone easily get started; if we put here all the considerations, it might be discouraging for the newcomer.